group-actors/src/main/java/org/sunbird/actors/CreateGroupActor.java

```diff
@@ -61,17 +61,15 @@ private void createGroup(Request actorMessage) throws BaseException {
     // adding members to group, if members are provided in request
     List<Map<String, Object>> reqMemberList =
         (List<Map<String, Object>>) actorMessage.getRequest().get(JsonKey.MEMBERS);
-    logger.info("Adding members to the group: {} groupId started", group.getName());
+    logger.info("Adding members to the group: {} groupId started", groupId);
     if (CollectionUtils.isNotEmpty(reqMemberList)) {
       memberList.addAll(reqMemberList);
     }
     Response addMembersRes =
         memberService.handleMemberAddition(
             memberList, groupId, requestHandler.getRequestedBy(actorMessage));
     logger.info(
-        "Adding members to the group : {} ended , response {}",
-        group.getName(),
-        addMembersRes.getResult());
+        "Adding members to the group : {} ended , response {}", groupId, addMembersRes.getResult());
```

Comment:
getResult() - wouldn't be too big to log this?

Reply:
No, it just, a success response